sdk/src/main/java/co/omise/android/models/Serializer.kt

```diff
@@ -98,6 +101,19 @@ class Serializer {
         return objectMapper.convertValue(model, object : TypeReference<Map<String, Any>>() {})
     }
 
+    /**
+     * Serializes the given [RequestBuilder] object to the provided output stream.
+     *
+     * @param outputStream The [OutputStream] to serialize the parameter into.
+     * @param builder      The [RequestBuilder] to serialize.
+     * @param <T>          The type of the parameter object to serialize.
+     * @throws IOException on general I/O error.
+    </T> */
+    @Throws(IOException::class)
+    fun <T : RequestBuilder<*>> serializeRequestBuilder(outputStream: OutputStream, builder: T) {
```

Comment:
Shall we use `Model` as specific generic type?

Reply:
I'll see if I can make it more specific 🙆‍♂